strawberryfields/engine.py

```diff
@@ -543,13 +542,12 @@ class RemoteEngine:
     >>> job = engine.run_async(program, shots=1)
     >>> job.status
     "queued"
-    >>> job.result
-    InvalidJobOperationError
-    >>> job.refresh()
+    >>> job.clear()
```

Comment:
Similar to the Sphinx documentation, I think [`xcc.Job.wait()`](https://xanadu-cloud-client.readthedocs.io/en/stable/api/xcc.Job.html#xcc.Job.wait) is more appropriate:
```suggestion
    >>> job.wait()
```